system/core/pages.php

```diff
@@ -195,6 +210,24 @@ function loadPage(RequestContext $O_O) {
 					exit;
 				}
 				break;
+
+				case HyphaRequest::HYPHA_SYSTEM_PAGE_HELP: //bz help
+				if ($args[0] == 'help') {
+							echo getButtonInfo($args);
+						exit;
+				} else {
+					$hyphaHtml->find('#pagename')->text( __('help-pagetitle')); //		"help-pagetitle" => "Help index",
+					$hyphaPage = new helpPage($args);
+					//echo "pages: 222<br>\n";
+					if (count($args) == 2) if (hypha_isLanguage($args[1])) {//echo " -pages.php 222- " . $args[1];
+						$hyphaHtml->writeToElement('langList', hypha_helpLanguages('',$args[1]));
```

Comment:
Why do you handle the language selection manually here? Does HyphaRequest not already handle that so you can just use `$O_O->getContentLanguage();` always? Or does it not handle languages for system pages, perhaps?

Reply:
The language for the help system differs from the languages for the pages. When the help index page is shown, the languages in the languages list are the languages from the help database.